browser/brave_shell_integration.h

```diff
@@ -10,6 +10,8 @@
 
 namespace shell_integration {
 
+void PinDefaultShortcutForExistingUsers();
```

Comment:
How about moving this into /brave_shell_integration_win.h for consistency?

Reply:
Moved. I thought this method could be cross platform.
but not sure it's available on macOS or linux.

Reply:
Oh, I see. We'll see if we can bump up things in shell_integration_win to abstract layer.